src/main/java/frc/robot/Controls.java

```diff
@@ -174,9 +179,42 @@ private void configureSuperStructureBindings() {
         .onTrue(Commands.runOnce(() -> branchHeight = BranchHeight.LEVEL_TWO).withName("level 2"));
     operatorController
         .a()
-        .onTrue(Commands.runOnce(() -> branchHeight = BranchHeight.LEVEL_ONE).withName("level 1"));
-    operatorController.rightBumper().onTrue(superStructure.stow().withName("Stow"));
-    driverController.a().onTrue(superStructure.intake());
+        .onTrue(Commands.runOnce(() -> branchHeight = BranchHeight.LEVEL_ONE).withName("level 1"))
+        .onTrue(
+            Commands.runOnce(() -> algaeIntakeHeight = AlgaeIntakeHeight.ALGAE_LEVEL_TWO_THREE)
+                .withName("algae level 2-3"));
+    ;
+
+    /*operatorController
+    .a()
+    .onTrue(Commands.runOnce(() -> branchHeight = BranchHeight.ALGAE_STOWED).withName("algae stow"));*/
+    operatorController
+        .leftTrigger()
+        .onTrue(
+            Commands.runOnce(() -> scoringMode = ScoringMode.ALGAE).withName("Algae Scoring Mode"));
+    operatorController
+        .rightTrigger()
+        .onTrue(
+            Commands.runOnce(() -> scoringMode = ScoringMode.CORAL).withName("Coral Scoring Mode"));
```

Comment:
Based on the OneNote, left and right trigger should both set coral scoring mode and left and right bumper should both set algae scoring mode.

Reply:
Did some more thinking about the controls and right bumper will be used for putting the elevator into coast, so we can keep it as is.